changelog/feat-google-play-test-mode-live-account-compatibility-notice

```diff
@@ -0,0 +1,4 @@
+Significance: patch
+Type: update
+
+feat: add compatibility notice for GooglePlay with live mode accounts.
```

Comment:
```suggestion
feat: add compatibility notice for GooglePay with live mode accounts.
```
This is really tiny and I almost hate even spotting it, but that should be Google**Pay**, no? 🤣